index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Raj Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  <title>Raj Portfolio</title>
+</head>
+ <body>
+  <main class="rj">
+   <header>
+      <a href="#"class="logo">My Logo</a>
+      <nav class="nav">
+          <div class="button"><a href="#"><img src="image/Union.png" alt="button"></a></div>
+      </nav>
+  </header>
+  <section>
+    <h1>Hey There. I'm Vivek Raj glad to see you!</h1>
```

Comment:
The size of this headline `<h1>Hey There. I'm Vivek Raj glad to see you!</h1>` in your design is `1.5em`, which is very small as compared to the one given in Figma design i.e. `font-size: 40px;`